helm/hpcc/values.yaml

```diff
@@ -34,6 +34,15 @@ global:
     resourceAttributes: # used to declare OTEL Resource Attribute config values
       deploymentEnvironment: development # used to anotate tracing spans' environment identifier (development/production/statiging/etc)
 
+  analyzerOptions:
+    disabled: false
+    analyzeInEclAgent: false
+    analyzeWhenComplete: false
```

Comment:
I think default values like this be commented out, to show that they are not needed, but left in for "documentation" purposes.